site/content/docs/developing/secrets.en.md

```diff
@@ -6,7 +6,7 @@ Secrets are sensitive bits of information like OAuth tokens, secret keys or API
 
 Adding secrets currently requires you to store your secret as a SecureString in [AWS Systems Manager Parameter Store](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-parameter-store.html) (SSM), then add a reference to the SSM parameter to your [manifest](../manifest/overview.en.md). 
```

Comment:
I know this isn't part of your PR, but now that I'm reading it, I wonder if "currently" seems odd/outdated here?